SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -61,47 +63,22 @@ namespace
 
 		// The format is expected to be:
 		// Function <objectName>::<functionName> queued on server/client for sending/execution for <duration> (and dropped). Reason: <reason>
```

Comment:
```suggestion
		// Function <objectName>::<functionName> sending/execution queued on server/client for <duration>. Reason: <reason>
```

Reply:
nit: I feel it reads slightly better this way

Reply:
Updated.